docs/ref/settings.txt

```diff
@@ -911,6 +911,74 @@ This is an Oracle-specific setting.
 
 The maximum size that the DATAFILE_TMP is allowed to grow to.
 
+.. setting:: DATAFILE_SIZE
+
+``DATAFILE_SIZE``
+^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+Default: ``'50M'``
+
+.. versionchanged:: 2.0
```

Comment:
I think the changed annotations/notes can be omitted. Most users are unlikely to care, I'd guess.